addons/stock_account/models/product.py

```diff
@@ -126,6 +126,8 @@ def _compute_value_svl(self):
             domain.append(('create_date', '<=', to_date))
         groups = self.env['stock.valuation.layer']._read_group(domain, ['value:sum', 'quantity:sum'], ['product_id'])
         products = self.browse()
+        # Browse all products and compute products' quantities_dict in batch.
```

Comment:
@Aurelienvd I would have made explicit that it's for having them cached, for perf. You're explaining what you do, but you miss the opportunity to explain the reason why you do it :)